frontend/src/redux/user/types.ts

```diff
@@ -7,6 +7,7 @@ export interface UserInfo {
   id: number;
   roles: RoadmapRole[];
   representativeFor?: Customer[];
+  defaultRoadmapId: number;
```

Comment:
This should include null
```suggestion
  defaultRoadmapId: number | null;
```